templates/homepage-banner-band.tmpl

```diff
@@ -34,8 +34,7 @@
       <div class="banner-welcome-image">
         <img src="/images/bullito_welcome.svg"
              alt="{{ homepage.banner.welcome_alt }}"
-             width="350"
-             height="300" />
+             />
```

Comment:
This is failing the linter for specifying height/width, is there a reason to delete it? (I.e does our linter need changes instead?)